ads/README.md

```diff
@@ -130,6 +130,22 @@ Here are some factors that affect whether the resize will be executed:
 - Whether the resize is requested for a currently active ad;
 - Whether the resize is requested for an ad below the viewport or above the viewport.
 
+
+### Support for variable size ad requests
```

Comment:
should we make the name of the feature consistent all over the doc? `variable size` => `multi-size`?